frontend/src/e2e-test/utils/page.ts

```diff
@@ -186,6 +186,18 @@ export class Element {
     return this.locator.textContent()
   }
 
+  get textContentAsFloat(): Promise<number | null> {
+    return this.locator
+      .textContent()
+      .then((str) => (str ? parseFloat(str) : null))
+  }
+
+  get textContentAsInt(): Promise<number | null> {
+    return this.locator
+      .textContent()
+      .then((str) => (str ? parseInt(str) : null))
+  }
```

Comment:
Is this ever really needed since javascript only has floats? In general, I personally prefer comparing strings since that's what the content is anyway.